binding.gyp

```diff
@@ -6,11 +6,12 @@
     {
       "target_name": "protagonist",
       "include_dirs": [
-        "./snowcrash/src",
-        "./snowcrash/ext/markdown-parser/src",
-        "./snowcrash/ext/markdown-parser/ext/sundown/src",
-        "./snowcrash/ext/markdown-parser/ext/sundown/html",
-        "./snowcrash/ext/sos/src",
+        "drafter/src",
```

Comment:
I mean they fail, because there in no `circle.yml` to provide "build recipe". Will backport from master branch. 

On local you can build w\o problems